src/client/mvt/feature-decoder.js

```diff
@@ -266,9 +266,27 @@ function clipLine(line) {
         point0 = nextPoint;
         type0 = nextType;
     }
+
+    clippedLine = removeDuplicatedVerticesOnLine(clippedLine);
     if (clippedLine.length > 0) {
         clippedLines.push(clippedLine);
     }
 
     return clippedLines;
 }
+
+function removeDuplicatedVerticesOnLine(line) {
```

Comment:
Minor thing: private functions start with `_` 😬The rest looks ✨ 👍

Reply:
Since when? :slightly_smiling_face:

Reply:
We have discussed this before many times. Private functions (both a method in a class and a function) starts with `_`. If it is going to be used outside, then it does not.

Reply:
Ok...